lib/kino_flame/runner_cell.ex

```diff
@@ -5,13 +5,35 @@ defmodule KinoFLAME.RunnerCell do
   use Kino.JS.Live
   use Kino.SmartCell, name: "FLAME runner cell"
 
-  @text_fields ["name"]
+  @text_fields ["name", "backend"]
   @number_fields ["min", "max", "max_concurrency"] ++ ["fly_cpus", "fly_memory_gb", "fly_gpus"]
+  @default_pod_template ~S"""
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    generateName: livebook-flame-runner-
+  spec:
+    containers:
+      - name: livebook-runtime
+        resources:
+          limits:
+            cpu: "1"
+            memory: 1Gi
```

Comment:
Looks like 1GB RAM is not enough to sync all the deps to the runner. I would set it to 4Gi, or perhaps remove the limits entirely and let the user set those if needed?

Reply:
you are right. We can let the people define resources. They should know best what they need.